actions/workflows/st2_package_publish_release.yaml

```diff
@@ -0,0 +1,77 @@
+---
+  chain:
+    -
+      name: "short_version"
+      ref: "core.local"
+      params:
+        cmd: "echo {{st2_release_version}} | cut -d \".\" -f1-2"
+      publish:
+        st2_release_base_ver: "{{short_version.stdout}}"
+        build_key: "st2_v{{st2_release_base_ver}}_build_number"
+      on-success: "get_current_build"
+    -
+      name: "get_build_server"
+      ref: "linux.dig"
+      params:
+        hostname: "st2-build-slave-ubuntu.service.consul"
+        rand: true
+        count: 1
+      on-success: "setup_st2client_rule_for_branch"
+    -
+      name: "setup_st2client_rule_for_branch"
+      ref: "st2cd.create_version_release_st2client_rule"
+      params:
+        hosts: "{{get_build_server.result[0]}}"
+        st2-base-url: "{{st2_base_url}}"
+        branch: "v{{st2_release_base_ver}}"
+      on-success: "setup_tag_rule_for_branch"
+    -
+      name: "setup_tag_rule_for_branch"
+      ref: "st2cd.create_version_tag_rule"
+      params:
+        hosts: "{{get_build_server.result[0]}}"
+        st2-base-url: "{{st2_base_url}}"
+        branch: "v{{st2_release_base_ver}}"
+      on-success: "setup_prod_pkg_rule_for_branch"
+    -
+      name: "setup_prod_pkg_rule_for_branch"
+      ref: "st2cd.create_prod_pkg_rule"
+      params:
+        hosts: "{{get_build_server.result[0]}}"
+        st2-base-url: "{{st2_base_url}}"
+        branch: "v{{st2_release_base_ver}}"
+      on-success: "run_ubuntu_prod_pkg_for_release"
+    -
+      name: "run_ubuntu_prod_pkg_for_release"
```

Comment:
Won't this kick off the packaging process again? Just wanna make sure you want these as part of this chain.